src/final2/subtests/MoveTest.java

```diff
@@ -12,16 +16,20 @@
  */
 public class MoveTest extends LangtonSubtest {
 
+	public MoveTest() {
+		setAllowedSystemExitStatus(SystemExitStatus.WITH_0);
+	}
+
 	/**
 	 * Asserts that {@code move} works on a very simple, easy to understand example: The rule is set to
 	 * {@code 90-90-90-90-90} and we have only one ant on this 2x2 board:
 	 * 
-	 * <pre>
 	 * <code>
+	 * <pre>
 	 * 0a
 	 * 00
-	 * </code>
 	 * </pre>
+	 * </code>
```

Comment:
> No, it should not. Because I can't convince the formatter not to break after <code>.

Some IDE features do not change specifications or semantics of a markup language